software/source/server/async_server.py

```diff
@@ -37,15 +35,24 @@
 )
 
 
+async def get_debug_flag():
+    return app.state.debug
+
+
 @app.get("/ping")
 async def ping():
     return PlainTextResponse("pong")
 
 
 @app.websocket("/")
-async def websocket_endpoint(websocket: WebSocket):
+async def websocket_endpoint(
+    websocket: WebSocket, debug: bool = Depends(get_debug_flag)
+):
     await websocket.accept()
 
+    # interpreter.tts set in the profiles directory!!!!
```

Comment:
Hey Kai! Why move this here? I think this might initialize the object / delete message history every time the websocket reconnects.